cdap-docs/admin-manual/source/cdap-hadoop-compatibility.rst

```diff
@@ -24,63 +24,65 @@ compatibility or know to be incompatible.
-| 3.3.0          | **No**  | **No**  | **No**  | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** |
-+----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
-| 3.2.x          | **No**  | **No**  | **No**  | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | **Yes** | **Yes** | **Yes** | **Yes** |
-+----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
-| 3.1.x          | **No**  | **No**  | **No**  | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | **Yes** | **Yes** | **Yes** | **Yes** |
-+----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
-| 3.0.x          | **No**  | **No**  | **No**  | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | **Yes** | **Yes** | **Yes** |
-+----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
-| 2.8.x          | **No**  | **No**  | **No**  | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    |
-+----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **CDH Distro Versions**                                                                                                                                              |
++================+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+
+| CDAP Version   | 5.15    | 5.14    | 5.13    | 5.12    | 5.11    | 5.10    | 5.9     | 5.8     | 5.7     | 5.6     | 5.5     | 5.4     | 5.3     | 5.2     | 5.1     |
++----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| 6.0.0          | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **No**  | **No**  | **No**  |
++----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| 5.1.0          | **No**  | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **No**  | **No**  | **No**  |
```

Comment:
do we need to add 5.1.1 and 5.1.2 ?

Reply:
yea, if there were no changes in any of 5.1.x, can add `.x` instead to be more general.

Reply:
i'd vote for listing them out for consistency, but 👍 either way

Reply:
I see 2 different formats. 
![image](https://user-images.githubusercontent.com/14131070/57896494-e85ba300-7805-11e9-8e4f-e8ffc1474771.png)

![image](https://user-images.githubusercontent.com/14131070/57896501-ed205700-7805-11e9-8dba-2f475e86162f.png)

But will use latest format.

Reply:
Fixed.